examples/CSharpDev/HttpTests/SimpleHttpTest.cs

```diff
@@ -28,11 +26,32 @@ public static void Run()
                 .WithWarmUpDuration(Seconds(5))
                 .WithLoadSimulations(
                     Simulation.InjectPerSec(rate: 1, during: TimeSpan.FromSeconds(30))
+                )
+                .WithThresholds(
+                    Threshold.RequestAllCount(x => x > 200),
+                    Threshold.RequestOkCount(x => x > 190),
+                    Threshold.RequestFailedCount(x => x <= 10),
+                    Threshold.RequestFailedRate(x => x < 0.1),
+                    Threshold.RPS(GetRpsConfig),
+                    Threshold.LatencyMin(x => x < 100),
+                    Threshold.LatencyMean(x => x < 400),
+                    Threshold.LatencyMax(x => x < 500),
+                    Threshold.LatencyStdDev(x => x is > 100 and < 200),
+                    Threshold.LatencyPercent50(x => x < 300),
+                    Threshold.LatencyPercent75(x => x < 320),
+                    Threshold.LatencyPercent95(x => x < 400),
+                    Threshold.LatencyPercent99(x => x < 500),
+                    Threshold.DataTransferAllBytes(x => x < 10000)
```

Comment:
I decided to flatten the thresholds model due to the following reasons:
- After renaming `Metric` to `Thresholds`, this list would contain `Threshold` list of `Thresholds`. This would look weird.
- It seems to me that in a real-world case, probably, just a few thresholds may be usually specified, so it would look very nested here.
- The flat list is simpler.

@AntyaDev Does this seem reasonable to you?

Reply:
yes, it looks good